.circleci/config.yml

```diff
@@ -5,21 +5,26 @@ orbs:
   gcp-cli: circleci/gcp-cli@1.8.4
 
 jobs:
-  build:
+  checkout:
     executor:
       name: go/default
       tag: '1.14'
     steps:
       - checkout
-      - restore_cache:
-          key: v1-repo-{{ .Environment.CIRCLE_BRANCH }}
-      - run:
-          name: Downloading Go modules
-          command: make mod-download
-      - save_cache:
-          key: v1-repo-{{ .Environment.CIRCLE_BRANCH }}
+      - go/mod-download-cached
```

Comment:
since the `go.sum` and `go.mod` files now exist in the repo root, we can use the function from the golang orb which  cleans up some lines